src/main/resources/META-INF/copyright.xml

```diff
@@ -0,0 +1,7 @@
+<idea-plugin>
+
+    <extensions defaultExtensionNs="com.intellij.copyright">
+        <updater filetype="D file"
+                 implementationClass="com.maddyhome.idea.copyright.psi.UpdateAnyFileCopyright$Provider"/>
```

Comment:
`com.maddyhome.idea.copyright.psi.UpdateAnyFileCopyright` doesn't seem like a class that ships with Intellij.  I was expecting a dlang specific implementation.

Reply:
This class comes from the copyright plugin itself. And the functionality is integrated only if the copyright plugin is here.